automation/taskcluster/lib/module_definitions.py

```diff
@@ -24,3 +24,14 @@ def from_gradle():
         'path': "{}/target.maven.zip".format(module['buildPath']),
         'shouldPublish': module['shouldPublish']
     } for module in gradle_modules]
+
```

Comment:
I'm not very proud of the naming of this function, nor this module. I'd rather move all these functions under `util` module. What do you think? Can do this either now or in a follow-up.

Reply:
Yeah, at least for this function, it doesn't quite make sense here: `module_definitions.from_gradle_version()` -> gives you a version string?

IIRC, I named the above function (`from_gradle()`) to take advantage of the name of the file to provide context for what the function would do. Unfortunately, since we aren't getting "module definitions" from any other source, it makes this file pretty empty, and also makes this file not very applicable for other gradle-related functions.

So, yeah, I definitely agree: let's check these into `utils` and update the function names accordingly :smile:

Reply:
How about `module_definitions.get_version_from_gradle()`? Or `module_definitions.get_version()`?

Reply:
Let's declare `get_version_from_gradle` the winner for now and I'll address in a follow-up the reorg of these methods within the `util.py` module, across all generation methods.